app/views/generic_files/_browse_everything.html.erb

```diff
@@ -4,6 +4,7 @@
 </div>
 <div class="well">
   <%= form_tag(sufia.generic_files_path, id: 'browse_everything_form', method: 'post') do %>
+      <%= render partial: 'generic_files/upload/to_collection' %>
```

Comment:
Can you confirm that proper place to put the config for hide/show upload to collection is in [sufia/models/engine.rb](https://github.com/projecthydra/sufia/blob/master/sufia-models/lib/sufia/models/engine.rb)?  If not, where should the configuration be defined?